components/database/sqlite/Makefile

```diff
@@ -23,16 +23,17 @@
 #
 
 BUILD_BITS= 64_and_32
+ENV=/usr/bin/env
```

Comment:
Please do not do this, unless _really_ needed.  If it is really needed, then please add a comment explaining why it is needed.  Thank you.

Reply:
In addition, it is a no-op because `ENV` is set unconditionally in `shared-macros.mk` anyway.  So please do not bother with a comment and simply remove this line.  Thank you.